02/app.ts

```diff
@@ -1,8 +1,67 @@
-type Student = {
-    // uzupełnij wedle uznania
+class UUID {
+    static count: number = 0;
+    static uuidMap: Map<string, string> = new Map(); 
```

Comment:
Raczej bym zapisywał tylko do sprawdzenia czy już taki uuid został wygenerowany, aby nie duplikować. To niestety też nie będzie idealne, bo mogliśmy już dany uuid wygenerować wczoraj lub miesiąc temu dlatego w calach "edukacyjnych", bym się tym nie przejmował. Można też np. wykorzystać pierwsze znaki na "datę", co dałoby jakąś pewność niepowtarzania.